test/parallel/test-net-connect-options.js

```diff
@@ -3,6 +3,19 @@ const common = require('../common');
 const assert = require('assert');
 const net = require('net');
 
+var assertTimeoutCalled = false;
```

Comment:
We don't normally need variables like in tests. Look in some of the other tests for `common.mustCall()`.